_specifications/lsp/3.18/language/signatureHelp.md

```diff
@@ -217,15 +227,24 @@ export interface SignatureHelp {
 	activeSignature?: uinteger;
 
 	/**
-	 * The active parameter of the active signature. If omitted or the value
-	 * lies outside the range of `signatures[activeSignature].parameters`
-	 * defaults to 0 if the active signature has parameters. If
-	 * the active signature has no parameters it is ignored.
+	 * The active parameter of the active signature.
+	 * 
+	 * If `null`, no parameter of the signature is active (for example a named
+	 * argument that does not match any declared parameters). This is only valid
+	 * if the client specifies the client capability
+	 * `textDocument.signatureHelp.noActiveParameterSupport === true`
+	 * 
+	 * If omitted or the value lies outside the range of
+	 * `signatures[activeSignature].parameters` defaults to 0 if the active
+	 * signature has parameters.
+	 * 
+	 * If the active signature has no parameters it is ignored.
+	 * 
 	 * In future version of the protocol this property might become
-	 * mandatory to better express the active parameter if the
-	 * active signature does have any.
+	 * mandatory (but still nullable) to better express the active parameter if
+	 * the active signature does have any.
```

Comment:
Please specify that `null` is only supported from @since 3.18.0 on forward

Reply:
I've added this to the comments, although I don't think I can use `@since` without it applying to the whole field?